src/connector.js

```diff
@@ -0,0 +1,769 @@
+		noResultTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">Aucun résultat</h2>
+				<p>Aucun résultat ne correspond à vos critères de recherche.</p>
+				<p>Suggestions:</p>
+				<ul>
+					<li>Assurez-vous que tous vos termes de recherches sont bien orthographiés </li>
+					<li>Utilisez de différents termes de recherche </li>
+					<li>Utilisez des termes de recherche plus généraux </li>
+					<li>Consultez les&nbsp;<a href="/fr/sr/tr.html"> trucs de recherche </a></li>
+					<li>Essayez la <a href="/fr/chaires-recherche/rechercher/recherche-avancee.html">recherche avancée</a></li>
+				</ul>
+			</section>`;
+	}
+	else {
+		noResultTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">No results</h2>
+				<p>No pages were found that match your search terms.</p>
+				<p>Suggestions:</p>
+
+				<ul>
+					<li>Make sure all search terms are spelled correctly</li>
+					<li>Try different search terms</li>
+					<li>Try more general search terms</li>
+					<li>Consult the&nbsp;<a href="/en/sr/st.html">search tips</a></li>
+					<li>Try the&nbsp;<a href="/en/sr/srb/sra.html">advanced search</a></li>
+				</ul>
+			</section>`;
+	}
+}
+
+if ( !resultErrorTemplateHTML ) {
+	if ( lang === "fr" ) {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				<p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+	else {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				<p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+}
+
+if ( !querySummaryTemplateHTML ) {
+	if ( lang === "fr" ) {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] résultats de recherche pour "%[query]"</h2>`;
+	}
+	else {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] search results for "%[query]"</h2>`;
+	}
+}
+
+if ( !didYouMeanTemplateHTML ) {
+	if ( lang === "fr" ) {
+		didYouMeanTemplateHTML = 
+			`<p tabindex="-1" class="h4 mrgn-lft-md did-you-mean">Rechercher plutôt <button class="btn-link p-0">%[correctedQuery]</button> ?</p>`;
+	}
+	else {
+		didYouMeanTemplateHTML = 
+			`<p tabindex="-1" class="h4 mrgn-lft-md did-you-mean">Did you mean <button class="btn-link p-0">%[correctedQuery]</button> ?</p>`;
+	}
+}
+
+if ( !noQuerySummaryTemplateHTML ) {
+	if ( lang === "fr" ) {
+		noQuerySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" class="h4">%[numberOfResults] résultats de recherche</h2>`;
```

Comment:
Is this should include a `role=status`?

Reply:
Not this one, because it'll only show up on search pages that show results right on page load, such as the Advanced search pages.